app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -95,7 +108,7 @@ def update_referer
       def ensure_manager!
         # Even though the user can view this admin set, they may not be able to view
         # it on the admin page.
-        authorize! :manage_any, AdminSet
+        authorize! :manage, @admin_set
```

Comment:
Checking for `:manage_any` is too permissive now that users other than admins can edit an admin set.